spec/machine/MachineExports.thy

```diff
@@ -67,32 +67,42 @@ requalify_consts
   timerPrecision
   max_time
   getCurrentTime_buffer
+  factor1
+  factor2
```

Comment:
Do we actually need to export them? I might have missed them, but I don't think I saw their names mentioned in the rest of the proof (which is a good thing).

Reply:
(We should be able to export eg `factor1_non_zero` without exporting `factor1`)

Reply:
I know that there is at least one lemma that is using at least one of these at the moment. But maybe a helper lemma I wrote later in the game could be used instead. I'll take another look

Reply:
Cool. If it is mentioned in a lemma statement, then I just missed it and no change needed.

Reply:
I was able to remove the references to `factor1` and `factor2` (these have been renamed, too) so I've removed these from the `requalify_consts`